src/mint_json.cr

```diff
@@ -40,7 +41,13 @@ module Mint
           "node" => node(exception),
         }
       end
+
       parse_root
+
+      # TODO: Enable when releasing 0.13.0!
+      # raise MintJsonMintVersionMissing, {
+      #   "contents" => @json,
+      # } unless mint_version_parsed
```

Comment:
I'd allow for missing `mint-version` and treat it as `*`, i.e. any version. In this way it'll be enforced only when set, giving an "escape hatch" when you don't want to specify the `mint-version`.

```suggestion
```

Reply:
Fair enough :slightly_smiling_face: